### ISSUE TITLE
Executor command option missing from Spark image

### ISSUE THREAD

**Issue by george-zubrienko** [bitnami/containers#4319]
### Name and Version

bitnami/spark:3.2.1

### What steps will reproduce the bug?

Follow the guide https://spark.apache.org/docs/3.2.1/running-on-kubernetes.html#cluster-mode and try to launch a job using Bitnami image.

Driver starts up normally but executors fail with `executor command not found` error.

### What is the expected behavior?

Spark submission with k8s master runs as expected.

### What do you see instead?

A running driver and failing executors.

### Additional information

Native spark image entrypoint has this option:

https://github.com/apache/spark/blob/master/resource-managers/kubernetes/docker/src/main/dockerfiles/spark/entrypoint.sh#L84-L101

**Comment by george-zubrienko** [comment]
just fyi, we have this working with a manual fix - combining bitnami entrypoint script with the missing piece from spark image:

```
  executor)
    shift 1
    CMD=(
      ${JAVA_HOME}/bin/java
      "${SPARK_EXECUTOR_JAVA_OPTS[@]}"
      -Xms$SPARK_EXECUTOR_MEMORY
      -Xmx$SPARK_EXECUTOR_MEMORY
      -cp '/opt/bitnami/spark/conf::/opt/bitnami/spark/jars/*'
      org.apache.spark.scheduler.cluster.k8s.KubernetesExecutorBackend
      --driver-url $SPARK_DRIVER_URL
      --executor-id $SPARK_EXECUTOR_ID
      --cores $SPARK_EXECUTOR_CORES
      --app-id $SPARK_APPLICATION_ID
      --hostname $SPARK_EXECUTOR_POD_IP
      --resourceProfileId $SPARK_RESOURCE_PROFILE_ID
      --podName $SPARK_EXECUTOR_POD_NAME
    )
    ;;

  *)
    # Non-spark-on-k8s command provided, proceeding in pass-through mode
    CMD=("$@")
    ;;
esac

echo ""
exec "${CMD[@]}"

```

**Comment by aoterolorenzo** [comment]
Hi @george-zubrienko,

I've been working in this issue this days and the fix is about to be release, but I can say the proposed fix included `-cp "$SPARK_CLASSPATH:$SPARK_DIST_CLASSPATH"` so your comment just saved me some headaches before testing it.

I will update and test the fix and will ping you when it will be released.

Thank you for the contribution!

**Comment by aoterolorenzo** [comment]
Hi again!

The fix is now released with #7579! Thanks for taking the time in submitting the issue and contribute to a solution.
